index.d.ts

```diff
@@ -219,7 +221,7 @@ declare module 'ical-generator' {
       type(): attendeeType;
       type(type: attendeeType): ICalAttendee;
       delegatedTo(): string | ICalAttendee;
-      delegatedTo(attendee: string | ICalAttendee);
+      delegatedTo(attendee: string | ICalAttendee):string | ICalAttendee; /** Explicit return type */
```

Comment:
`delegatedTo` doesn't seem to return a string unless `attendee` is `undefined` which is covered by the overload just above it.